Tribler/Test/Core/CreditMining/test_creditmining_sys.py

```diff
@@ -20,8 +20,9 @@
 from Tribler.Main.Utility.GuiDBTuples import CollectedTorrent
 from Tribler.Policies.BoostingManager import BoostingManager, BoostingSettings
 from Tribler.Test.Core.CreditMining.mock_creditmining import MockLtTorrent, ResourceFailClass
-from Tribler.Test.common import TORRENT_FILE
-from Tribler.Test.test_as_server import TestAsServer, TESTS_DATA_DIR
+from Tribler.Test.Core.Modules.Channel.base_test_channel import BaseTestChannel
```

Comment:
Here you are creating a dependency again between `Tribler.Test.Core.Modules.Channel` and `Tribler.Test.Core.CreditMining`. Please try to avoid that.